### PR TITLE
awesome books title

### PR DESCRIPTION
**Pull request by bronnie313**
- Added an array of books
- Added a function to add books using the title and the author
- added a function to remove the books with the remove button
- made sure that the book is stored using local storage in the browser's memory

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -5,25 +5,24 @@ class Book {
   }
 }
 const addBook = document.getElementById('addBook');
```
Comment (topeogunleye) [inline script.js:7]: - [ ] I kindly suggest that you fix your addbook and removeBook functionality, when I add a book, it should be displayed oh the page, and stored in local storage and also when I remove a book, it should be removed from the UI and localstorage. Currently I'm unable to add a book. please fix this. Kindly check the video below for more explanation:


https://www.loom.com/share/969b6b2fa1d34207ac17cbb0611263f3

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @sawmon71293, @bronnie313,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good summary description :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (sawmon71293) [reply script.js:7]: I made sure the code works. But I don't know what's the problem is 
It worked on my side. 
Check below video 
@topeogunleye  
https://www.loom.com/share/b9728421edf843ca94889e12cf9bc5f1

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @sawmon71293, and @bronnie313👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**